qt/scientific_interfaces/Inelastic/Analysis/FqFitDataPresenter.cpp

```diff
@@ -246,6 +253,21 @@ void FqFitDataPresenter::addWorkspace(const std::string &workspaceName, const st
   }
 }
 
+std::map<std::string, std::string> FqFitDataPresenter::chooseFqFitFunctions(bool paramWidth) {
+  if (m_view->isTableEmpty()) { // when first data is added to table, it can only be either WIDTH or EISF
+    return paramWidth ? FqFit::WIDTH_FITS : FqFit::EISF_FITS;
+  } else {
```

Comment:
Nice use of a ternary operator 👍 In this case we technically don't need the `else` because this function will return anyway if we enter the if statement. Sometimes not using an `else` can mean we can de-dent some of our code and make it more readable

Reply:
Thanks for letting me know. I agree is not necessary, but I would have though is better to have in this case.